tox.ini

```diff
@@ -1,2 +1,17 @@
 [flake8]
 max-line-length=88
+exclude=.cache,.venv,.git
+docstring-convention=all
+import-order-style=pycharm
+ignore=
+    B311,W503,E226,S311,T000
+    # Missing Docstrings
+    D100,D104,D105,D107,
+    # Docstring Whitespace
+    D203,D212,D214,D215,
+    # Docstring Quotes
+    D301,D302,
+    # Docstring Content
+    D400,D401,D402,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D416,D417
+    # Type Annotations
+    ANN002,ANN003,ANN101,ANN102,ANN204,ANN206
```

Comment:
Not sure we should ignore all of these, in fact, I think most of these should be kept (I'll happily fix any linting errors this causes!)

Reply:
Can you remove from list what you want yourself? I don't know what to remove and what to leave.

Reply:
```suggestion
    # Todo comments
    T000
    # Docstring Whitespace
    D203,D212,D214,D215,
    # Docstring Quotes
    D301,D302,
    # Docstring Content
    D400,D401,D402,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D416,D417
```